examples/runnable-example/README.md

```diff
@@ -0,0 +1,41 @@
+
+# Runnable Example
+This is an eample of how to run Consul-Terraform-Sync with Consul. It uses the Terraform provider [local](https://registry.terraform.io/providers/hashicorp/local/latest), which manages local resources. In this case, it will be used to create local files.
+
+This example is composed of:
+- Consul-Terraform-Sync configuration
+- Terraform module that is Consul-Terraform-Sync compatible
+- Consul configuration
+
+## Demo
+
+Start Consul agent in dev mode, using the provided Consul config directory. This will register multiple services to Consul.
+
+```
+$ consul agent -dev -config-dir=./consul.d
```

Comment:
Yeah, I was working under the assumption that they'd be on the path since I had followed the installation steps in the CTS README. Is there an advantage to having the user download it to this working directory instead? Otherwise, I'll put the explicit instructions under the requirements that CTS and Consul are installed in their path.

Reply:
Ah! I see. I think that having the users go through the installation steps in cts readme makes sense! I just didn't realize this assumption. Let me resolve this comment and update the one I have about steps for installing Consul + CTS. Thanks for clarifying!